api/shipping_categories.rst

```diff
@@ -0,0 +1,421 @@
+Shipping categories API
+=======================
+
+These endpoints will allow you to manage shipping categories via simple crud. Endpoints base is `/api/shipping-categories`.
```

Comment:
``will allow``? or ``allows``? They should allow whatever they allows in time of reading this sentence :)